### PR TITLE
ExportMfpPlugin: add failure handling

### PR DESCRIPTION
**Pull request by taulinger**

### CODE REVIEW

File: test/plugins/ExportMfpPlugin.test.js [hunk 1]
```diff
@@ -61,9 +83,52 @@ describe('ExportMfpPlugin', () => {
...
+			// const errorSpy = spyOn(console, 'error');
+
+			// startJob(spec);
+
+			// expect(store.getState().mfp.jobSpec.payload).not.toBeNull();
+			// await TestUtils.timeout();
+			// expect(store.getState().mfp.jobSpec.payload).toBeNull();
+			// expect(store.getState().notifications.latest.payload.content).toBe('exportMfpPlugin_mfpService_createJob_exception');
+			// expect(store.getState().notifications.latest.payload.level).toBe(LevelTypes.ERROR);
+			// expect(errorSpy).toHaveBeenCalledWith('PDF generation was not successful.', jasmine.anything());
```
Comment (thiloSchlemmer) [inline test/plugins/ExportMfpPlugin.test.js:127]: dead code

Comment (taulinger) [reply test/plugins/ExportMfpPlugin.test.js:127]: oops

Comment (taulinger) [reply test/plugins/ExportMfpPlugin.test.js]: Done

**Review by thiloSchlemmer — APPROVED** [review]
Well done!

**Pull request merged**